app/tests/LocationTest.kt

```diff
@@ -0,0 +1,11 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class LocationTest {
+    @Test
+    fun `test location`() {
+        val location = Location(12.51, -50.12)
+        assertEquals(12.51, location.x)
+        assertEquals(-50.12, location.y)
```

Comment:
I don't think `x` and `y` are bad given that the specification uses `x` and `y` and this notation is used often by programmers for various points. `longitude` and `latitude` could be a better choice for a location, but I'd keep `x/y` to be consistent with the docs.

Reply:
One exception could be the usage output: `Usage: <program name> <user x coordinate> <user y coordinate> <shop data filename>.`

I think it would be easier to understand this message with `longitude/latitude` if you are not a developer.